lib/bolt/executor.rb

```diff
@@ -232,6 +233,11 @@ def report_bundled_content(mode, name)
       @analytics.report_bundled_content(mode, name)
     end
 
+    def report_file_source(plan_function, source)
+      label = Pathname.new(source).absolute? ? 'absolute' : 'module'
+      @analytics&.event('Plan', plan_function, label: label)
```

Comment:
This is a little weird because it means we'll get both "Plan / call_function / file::read" and "Plan / file::read / absolute". That namespacing seems a bit off, but I'm not sure there's a better option...